### ISSUE TITLE
different result on cuda8 and cuda9

### ISSUE THREAD

**Issue by bobwan1995** [rowanz/neural-motifs#5]
Hi Rowan, I'm running your codes now, but I found I'll get totally different result on cuda8 and cuda9 when I train detector(I don't use lstm here). Results from cuda9 are reasonable, while the loss from cuda8 will not decrease after several iterations. Could you please tell me why it happens and I can solve it? I need to put the project to a server with cuda8 and I can't change cuda version for that server. Thank you!

**Comment by rowanz** [comment]
sorry for the late response - the short answer is that I'm not sure. I was able to get my code running on a server that only has Cuda8.0 installed, so I suspect it might not be the cuda installation. Are you sure you've set everything up correctly (using the same pretraining as I did, etc)?

**Comment by bobwan1995** [comment]
I'm sorry for my issue, I found the problem was mainly caused by the different batch size I set. If batch size is large the model wouldn't converge at the first epoch.  Thank you a lot!
